src/oidc/authorize.ts

```diff
@@ -96,7 +124,7 @@ export const handler = async (
     );
   }
   try {
-    await writeNonce(code, nonce);
+    await writeNonce(code, nonce, cookies?.userId);
```

Comment:
If I understand optional chaining correctly, the user ID parameter will be `undefined` if the cookie's not set. Is Typescript smart enough to not override the default user ID in that case?

Reply:
Yes, I just tested it with this script

```
const testFn = (foo = "bar") => {
  console.log(foo);
};

const hello = {};

testFn(hello?.world);

//output = "bar"
```